simulator/src/lib.rs

```diff
@@ -23,8 +23,13 @@
 //!
 //! ## Windows
 //!
-//! The Windows install process is a bit more involved, but it _does_ work. See [the SDL2
-//! wiki](https://wiki.libsdl.org/Installation#WinRT.2FWindows_8.2FWinPhone) for instructions.
+//! The Windows install process is a bit more involved, but it _does_ work. See [the Rust-SDL2
+//! crate's README](https://github.com/Rust-SDL2/rust-sdl2) for instructions. There are multiple
+//! ways to get it working, but probably the simplest method is copying the binaries as shown
+//! [here](https://github.com/Rust-SDL2/rust-sdl2#windows-msvc).
+//!
+//! **Note:** The precompiled binaries need to be added to _all_ Rust versions you use. For example,
+//! this means that you will need to copy the files after every update to a new stable Rust version.
```

Comment:
I think this can be removed as it's outside the simulator setup instructions and should be covered by the SDL2 side.

Reply:
Fair enough